library/Zend/Amf/Parse/Serializer.php

```diff
@@ -44,6 +44,8 @@ abstract class Zend_Amf_Parse_Serializer
      */
     protected $mbStringFunctionsOverloaded;
 
+    protected $_mbStringFunctionsOverloaded;
```

Comment:
two very similar properties: $mbStringFunctionsOverloaded, $_mbStringFunctionsOverloaded. previous rename but incorrectly?

Reply:
- Correct property name is `$_mbStringFunctionsOverloaded`
- `$mbStringFunctionsOverloaded` is typo from Rob Allen, it's ok with php < 8.2 bug will raise 'deprecated' on php 8.2

https://github.com/Shardj/zf1-future/commit/5a9cf80e4646667e3b4be18c3fedd0e4b2403287#diff-b2f175f1769a1de6580426d48387c7977c6cf21685557f608aaef3a116f82b71

Reply:
so, remove the typo?